prql-compiler/src/sql/translator.rs

```diff
@@ -99,40 +96,24 @@ pub fn translate_query(query: &Query) -> Result<sql_ast::Query> {
     Ok(main_query)
 }
 
-struct AtomicTable {
+pub struct AtomicTable {
     name: String,
-    frame: MaterializedFrame,
-    pipeline: Vec<Transform>,
+    pipeline: Pipeline,
+    frame: Option<MaterializedFrame>,
 }
 
-pub fn load_std_lib() -> Result<Vec<Node>> {
-    use crate::parse;
-    let std_lib = include_str!("./stdlib.prql");
-    Ok(parse(std_lib)?.nodes)
-}
-
-fn separate_pipeline(query: &Query) -> Result<(Vec<Table>, Vec<Node>, Vec<Pipeline>)> {
+fn into_tables(nodes: Vec<Node>) -> Result<Vec<Table>> {
```

Comment:
The convention has been `tables_of_nodes`, though no strong view on the exact convention; it is nice to have one though